sdk/include/hipdnn_sdk/test_utilities/FlatbufferDatatypeMapping.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <hipdnn_sdk/data_objects/data_types_generated.h>
+#include <hipdnn_sdk/utilities/UtilsBfp16.hpp>
+#include <hipdnn_sdk/utilities/UtilsFp16.hpp>
+
+namespace
```

Comment:
I don't think an unnamed namespace in a header file is a common pattern. Unnamed namespaces limit linkage to file-scope with cpp files (similar to the static keyword), but for a header file the "file-scope" is determined by how it's included. You really want to use a project namespace.